syntax = "proto3";


// Message used to store the request 
// data in the multiple messages
message RequestInfo{
    string request_id = 1;
    string worker_id = 2;
}